utils/interfaces/_open_telemetry.py

```diff
@@ -26,15 +26,17 @@ def ingest_file(self, src_path):
         return super().ingest_file(src_path)
 
     def get_otel_trace_id(self, request):
-        paths = ["/api/v0.2/traces"]
+        paths = ["/api/v0.2/traces", "/v1/traces"]
         rid = get_rid_from_request(request)
 
         if rid:
             logger.debug(f"Try to find traces related to request {rid}")
 
         for data in self.get_data(path_filters=paths):
             export_request = ExportTraceServiceRequest()
-            content = eval(data["request"]["content"])  # Raw content is a str like "b'\n\x\...'"
+            raw_content = data["request"]["content"]
+            # Raw content can be either a str like "b'\n\x\...'" or bytes
+            content = eval(raw_content) if isinstance(raw_content, str) else raw_content
```

Comment:
This logic must be done in `utils/interfaces/_deserializer.py::deserialize_http_message`

Reply:
Got it, moved it there.